### PR TITLE
erc20-token-and-lp-weighted strategy [erc20-token-and-lp-weighted]

### PR DESCRIPTION
**Pull request by joehquak**
Changes proposed in this pull request:
- Introduction of `erc20-token-and-lp-weighted` strategy

Screenshot 2022-07-25 at 20 42 20

### CODE REVIEW

**Comment by joehquak** [comment]
@ChaituVR :)

**Comment by joehquak** [comment]
@ChaituVR hope this PR is suitable

Please let me know if you have any requests 🙏

**Comment by joehquak** [comment]
@ChaituVR please let me know if there are any issues with this PR 🙏

File: src/strategies/erc20-token-and-lp-weighted/index.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, multicall } from '../../utils';
+
+export const author = 'joehquak';
+export const version = '0.1';
```
Comment (ChaituVR) [inline src/strategies/erc20-token-and-lp-weighted/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**